python/ray/dashboard/subprocesses/utils.py

```diff
@@ -43,3 +46,18 @@ def get_socket_path(socket_dir: str, module_name: str) -> str:
 
 def get_named_pipe_path(module_name: str) -> str:
     return r"\\.\pipe\dash_" + module_name
+
+
+def get_http_session_to_module(
```

Comment:
this can be used by the `handle.py` as well?

Reply:
done